runtime/testy/runtime.py

```diff
@@ -200,16 +221,19 @@ def runtimeTest(testNames):
   for testName in testNames:
     testFileName = "%s_output" % (testName,)
     with open(testFileName, "w", buffering = 1) as testOutput:
+      print("Running test: {}".format(testName), end="")
+      sys.stdout.flush()
```

Comment:
FYI this may be equivalent to doing `print("...", flush=True)` on the line above.